From fcfea8d385d995f1eb46fd5585dcba13967ce236 Mon Sep 17 00:00:00 2001
From: Fabien Dessenne <fabien.dessenne@foss.st.com>
Date: Wed, 17 Nov 2021 14:47:47 +0100
Subject: [PATCH 26/49] perf: stm32: update build dependency

Build this driver, not only for STM32MP15, but for all STM32.

Upstream-Status: Downstream [https://github.com/STMicroelectronics/linux.git v5.15-stm32mp 024f17f119f4d2600b4ef6ef92aab7a48180cf4f]
Signed-off-by: Fabien Dessenne <fabien.dessenne@foss.st.com>
Change-Id: I840d2313b5b27bf45f639e89a3c339b7c731233e
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/228520
Reviewed-by: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Reviewed-by: Alexandre TORGUE <alexandre.torgue@foss.st.com>
---
 drivers/perf/Kconfig | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/perf/Kconfig b/drivers/perf/Kconfig
index e09335c1612d9..2c908b01428c0 100644
--- a/drivers/perf/Kconfig
+++ b/drivers/perf/Kconfig
@@ -139,7 +139,7 @@ config QCOM_L3_PMU
 
 config STM32_DDR_PMU
 	tristate "STM32 DDR PMU"
-	depends on MACH_STM32MP157
+	depends on ARCH_STM32
 	default m
 	help
 	  Support for STM32 DDR performance monitor (DDRPERFM).
-- 
2.39.2

